### DISCUSSION TITLE
Steps 组件

### DISCUSSION THREAD

**Discussion by yangjinjun3** [jdf2e/nutui-react#24] [General]
功能、基本交互测试
...
文档完整性验证
...
...
props 所有属性测试
1、无法自定义class和style
...
props 异步修改组件是否动态更新
...
...
emits 所有事件测试
...
...
npm run build 子组件是否被构建
...
其它

**Comment by dushoujun** [comment]
1. className和style props缺失
2. step.tsx中className命名没有使用bem方式

**Comment by junjun666** [comment]
上述问题已修复
